packages/ionic/src/commands/generate.ts

```diff
@@ -35,6 +35,20 @@ The given ${chalk.green('name')} is normalized into an appropriate naming conven
       description: 'The name of the component being generated',
     }
   ],
+  options: [
```

Comment:
You can provide examples of these options using the `exampleCommands` list above.